src/components/AddBook.jsx

```diff
@@ -1,19 +1,21 @@
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
 import { addBook } from '../redux/books/books';
+import style from '../styling/AddBook.module.css';
 
 function AddBook() {
   const [title, setTitle] = useState();
   const [author, setAuthor] = useState();
   const dispatch = useDispatch();
 
   return (
-    <div>
-      <h2>ADD NEW BOOK</h2>
-      <form>
-        <input type="text" value={title || ''} placeholder="Book title" onChange={(e) => setTitle(e.target.value)} required />
-        <input type="text" value={author || ''} placeholder="Author" onChange={(e) => setAuthor(e.target.value)} required />
+    <div className={style.AddBook}>
+      <h2 className={style.AddBook__title}>ADD NEW BOOK</h2>
+      <form className={style.AddBook__form}>
+        <input className={`${style.AddBook__input} ${style.AddBook__inputTitle}`} type="text" value={title || ''} placeholder="Book title" onChange={(e) => setTitle(e.target.value)} required />
+        <input className={`${style.AddBook__input} ${style.AddBook__inputAuthor}`} type="text" value={author || ''} placeholder="Author" onChange={(e) => setAuthor(e.target.value)} required />
```

Comment:
- [ ] Please, consider adding the "Category" choosing possibility, as mentioned in the original design.
- [ ] Additionally, please, consider to not forget to place the category name on top of the title, as also mentioned on the screen below

> Original 
<img width="705" alt="Screen Shot 2022-04-01 at 00 17 10" src="https://user-images.githubusercontent.com/49064106/161142866-4cfb6f4a-8ff5-4997-8133-a41687448504.png">